### DISCUSSION TITLE
Launch from Taskbar?

### DISCUSSION THREAD

**Discussion by 3Diva** [MoonRide303/Fooocus-MRE#97] [General]
I didn't know where to ask this, please forgive me if this isn't the place for it but is there a way to add a quick launch option to launch Fooocus-MRE from the taskbar? Or even just a way to create a desktop icon that will launch Fooocus-MRE? I've figured out how to make desktop icons to launch Autotmatic1111 as well as ComfyUI but I can't seem to figure it out for Fooocus. It's a bit of a pain to dig into my folders to get to the launch file. I know it's not a big emergency or a big deal, it would just be helpful to be able to launch it from the taskbar or even the desktop. Thank you in advance to whomever can give me some help with this! :D

**Comment by barepixels** [comment]
Right click "run-mre.bat" create a shortcut
Change the shortcut file name to Fooocus-MRE
Move the Fooocus-MRE shortcut to:
C:\Users\barepixels\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\AI Softwares

![zzGreenshot](https://github.com/MoonRide303/Fooocus-MRE/assets/92839033/7d804bea-0c81-4b1b-8e68-86f579967049)

**Comment by 3Diva** [comment]
Oh thank you, @barepixels ! I will give that a shot! I appreciate the help!
